spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -1466,7 +1466,7 @@ class CometExecSuite extends CometTestBase {
         SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> aqeEnabled) {
         withParquetTable((0 until 10).map(i => (i, 10 - i)), "t1") { // TODO: test nulls
           val aggregateFunctions =
-            List("MAX(_1)", "MIN(_1)") // TODO: Test all the aggregates
+            List("COUNT(_1)", "MAX(_1)", "MIN(_1)") // TODO: Test all the aggregates
```

Comment:
Is `count(*)` also valid in this context? Could you add a test if it is

Reply:
Added. Thanks